<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeStatusProjectDialogClass</class>
 <widget class="QDialog" name="ChangeStatusProjectDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeStatusProjectDialog</string>
  </property>
  <widget class="QLabel" name="nameLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменение статуса проекта</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата ахивирование проекта</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="desc">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkArchiveTask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Архивировать задачи</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="statusBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>561</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Закрыт успешно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Провален</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Приостановлен</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
